### ISSUE TITLE
Docs: states.file.managed location of non absolute paths

### ISSUE THREAD

**Issue by guettli** [saltstack/salt#28244]
https://docs.saltstack.com/en/latest/ref/states/all/salt.states.file.html#salt.states.file.managed

The docs for "name" are up to now: "The location of the file to manage"

Please add a note about relative paths in "name". What are they relative to?

**Comment by jfindlay** [comment]
@guettli, thanks for the report.

**Comment by twellspring** [comment]
Based on results of the below test, relative paths do not appear to be supported in the name field.  

```
----------
          ID: test_file_create
    Function: file.managed
        Name: ./zzzzz
      Result: False
     Comment: Specified file ./zzzzz is not an absolute path
     Started: 20:45:46.488474
    Duration: 1.075 ms
     Changes:
```

**Comment by rallytime** [comment]
Closed via #43290
